### DISCUSSION TITLE
some questions

### DISCUSSION THREAD

**Discussion by Angel0726** [AppFlowy-IO/AppFlowy#456] [General]
Data Compatibility: Will the new version be backward compatible with software updates? 

Profitable: Is it fully open source? How to make profit in the later stage? Are you making money with plugins?Will it be charged by restricting blocks like notion?

**Comment by lawweiliang** [comment]
Monetization part can refer to to this discussion thread
https://github.com/AppFlowy-IO/AppFlowy/discussions/15
